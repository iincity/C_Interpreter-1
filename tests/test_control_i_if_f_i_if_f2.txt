

int
a , b , c, d ;
{

c = 4;
d = 5;
c = c + d;
d = c + d;
if d > c {
	print a;
	print b;
	print c;
	print d;
	if a > c {
		a = a+1;
		b = b+1;
		print a;
		print b;
	}
	print a;
	print b;
	if c > a { 
		a = a+2;
		b = b+3;
		print a;
		print b;
	}
	a = a+1;
	print a;
}
a = 1 ;
b = 2 ;
if b > a {
	print a;
	print b;
	print c;
	print d;
}
	
 
if  b > a {
        a = a+b;
	print a;
	if b < a { 
		print a;
		b = a+b;
		if b > a {
			print a ;
			print b ;
			if b < a {
				print a;
				print b;
			}
		}
	}
	a = a+b;
	print a;
} 
a = a*b;
print a;
print b;
if a > b {
	print a;
	b = 1;
	a = a+b;
	print a;
	print b;
}

}
