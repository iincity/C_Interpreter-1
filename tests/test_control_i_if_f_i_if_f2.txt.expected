Parse Error: return statement with no expression in a function with a return type
Got ; at line 72, position 8
